docs/resources/routing_queue.md

```diff
@@ -172,26 +170,6 @@ Required:
 - `service_level_percentage` (Number) The desired Service Level. A float value between 0 and 1.
```

Comment:
Is the documentation generated off the comments inside the code.  Wouldnt you need to make the modifications there?